cmd/cmd.go

```diff
@@ -122,11 +128,29 @@ kubectl neat get -- svc -n default myservice --output json`,
 		if err != nil {
 			return err
 		}
-		cmd.Println(string(out))
+		err = printOutput(cmd, out)
+		if err != nil {
+			return err
+		}
 		return nil
 	},
 }
 
+func printOutput(cmd *cobra.Command, out []byte) error {
```

Comment:
instead of passing the entire cmd, can you please in just `cmd.OutOrStdout()` as `io.writer`?